apps/frontend/src/components/Process/Accordion/index.tsx

```diff
@@ -203,11 +209,15 @@ const AccordionProcess = (props: AccordionProcessProps) => {
           onClose={() => setShowEditProcessModal(false)}
           isOpen={showEditProcessModal}
           submit={async (values: ProcessFormValues) => {
+            let removedDisclosures = disclosures.filter(d => !values.disclosures.map(value => value.processIds).includes(d.processIds))
+            let addedDisclosures = values.disclosures.filter(d => !disclosures.map(value => value.processIds).includes(d.processIds))
+            addedDisclosures.forEach(d => updateDisclosure(convertDisclosureToFormValues({...d, processIds: [...d.processIds, currentProcess.id]})))
+            removedDisclosures.forEach(d => updateDisclosure(convertDisclosureToFormValues({...d, processIds: [...d.processIds.filter(p => p !== currentProcess.id)]})))
             await submitEditProcess(values) ? setShowEditProcessModal(false) : setShowEditProcessModal(true)
```

Comment:
denne logikken bør ligge i submitEditProcess, som du har gjort for create process